examples/asr/asr_webapp/README.rst

```diff
@@ -0,0 +1,70 @@
+**ASR SERVICE **
+================
+
+Usage
+-----
+
+A simple service that will load a pre-trained model from NeMo's ASR collection and after uploading some files, will transcribe them. Any pre-trained model can be selected and if the local / docker environment supports GPUs then it will be possible to transcribe the audio segments with GPU.
```

Comment:
They can also upload and perform inference on their own pre-trained model, yes?

Reply:
Ah good catch